rpc/src/rpc_pubsub.rs

```diff
@@ -640,8 +640,7 @@ mod tests {
             transaction::{self, Transaction},
         },
         solana_stake_program::stake_state,
-        solana_vote::vote_transaction::VoteTransaction,
```

Comment:
this breaks rpc consumers right because the enum layout is different?

Reply:
seems fine because the VoteTransaction is not directly communicated back to the consumer, it's always converted to an intermediary `RpcVote` https://github.com/anza-xyz/agave/blob/master/rpc/src/rpc_subscriptions.rs#L830-L836. 

Unless I'm missing somewhere else it's used?

Reply:
I don't believe so :)